js/csv_to_html_table.js

```diff
@@ -29,6 +31,22 @@ CsvToHtmlTable = {
                 for (var headerIdx = 0; headerIdx < csvHeaderRow.length; headerIdx++) {
                     $tableHeadRow.append($("<th></th>").text(csvHeaderRow[headerIdx]));
                 }
+
+                if (column_filters) {
+                    var $tableHeadRow_filter = $("<tr></tr>");
+                    for (var headerIdx = 0; headerIdx < csvHeaderRow.length; headerIdx++) {
+                        $tableHeadRow_filter.append($("<th></th>").html(
+                            '<input type="text" id="col_f-' + headerIdx + '" placeholder="filter column" />'));
```

Comment:
this code uses Bootstrap, so we should add the `form-control` and `form-control-sm` classes. Currently the default styles don't resize to fit the screen width and have a different aesthetic. 
![Screen Shot 2023-03-02 at 1 55 31 PM](https://user-images.githubusercontent.com/919583/222538065-44d10377-fb1d-40cd-9bca-487fef25bcd4.png)